tests/unit/connector/test_http_input.py

```diff
@@ -3,17 +3,42 @@
 # pylint: disable=attribute-defined-outside-init
 import multiprocessing
 from copy import deepcopy
+from unittest import mock
 
 import falcon
+import pytest
 import requests
 import uvicorn
+from requests.auth import HTTPBasicAuth
 
 from logprep.abc.input import FatalInputError
 from logprep.connector.http.input import HttpConnector
 from logprep.factory import Factory
 from tests.unit.connector.base import BaseInputTestCase
 
 
+@pytest.fixture(scope="function")
+def create_credentials(tmp_path_factory):
+    tmp_path = tmp_path_factory.mktemp("data")
+    secret_file_path = tmp_path / "secret-0.txt"
+    secret_file_path.write_text("secret_password")
+    credential_file_path = tmp_path / "credentials.yml"
+    credential_file_path.write_text(
+        f"""---
+input:
+  endpoints:
+    /auth-json-secret:
+      username: user
+      password_file: {secret_file_path}
+    /auth-json-file:
+      username: user
+      password: file_password
+"""
```

Comment:
```suggestion
    /.*/[A-Z]{{2}}/json$:
      username: user
      password: password
```

* regex endpoint for usage below